dbt_superset_lineage/push_descriptions.py

```diff
@@ -63,7 +63,7 @@ def get_datasets_from_superset(superset, superset_db_id):
             break
 
     assert datasets, "There are no datasets in Superset!"
-
+    
```

Comment:
Would you mind removing the extra whitespaces to keep it aligned with the rest of the code?
```suggestion

```

Reply:
Same below.

Reply:
Could you please also remove the added whitespace from the rest of your changes?